src/tests/ftest/pool/create_capacity.py

```diff
@@ -37,7 +37,11 @@ def test_create_pool_quantity(self):
         # available capacity, e.g. 0.6% for 100 pools.
         quantity = self.params.get("quantity", "/run/pool/*", 1)
         self.add_pool_qty(quantity, create=False)
+        # for multiple pool creation cases, enabling and disabling
+        # logmask setting to DEBUG explicitly to save run time..
+        self.dmg.server_set_logmasks("DEBUG", raise_exception=False)
```

Comment:
I think this line would need to be placed before the self.add_pool_qty() call above that will actually be doing the pool creates

Reply:
`self.add_pool_qty(quantity, create=False)` is not creating the pool yet at this stage as we have `create=False`. It is just creating different TestPool objects at this stage for the quantify specified. In the next step i.e `self.check_pool_creation(30)` the pool actually gets created using the multiple TestPool objects and verified if pool creation was completed within the expected 30 sec time limit.

Reply:
ah yes, thank you for correcting me on that detail that I missed.